Kernel/Syscalls/fork.cpp

```diff
@@ -153,8 +153,6 @@ ErrorOr<FlatPtr> Process::sys$fork(RegisterState& regs)
         }
     });
 
-    PerformanceManager::add_process_created_event(*child);
-
```

Comment:
Similar reason as to the `Process::register_new()` change here. This causes us to send the process created event a lot earlier, but `fork()`ing might still fail after that, leading us to emitting an event for a process that never existed.

Reply:
(Maybe we should add some comments that we should only send this event and register processes after we are sure that these functions won't fail. Just an idea; you don't need to do that in this PR of course)